src/constants.js

```diff
@@ -1,52 +1,25 @@
-Sk.builtin.str.$emptystr = new Sk.builtin.str("");
 
-/**
- * Python bool True constant.
- * @type {Sk.builtin.bool}
- * @memberOf Sk.builtin.bool
- */
-Sk.builtin.bool.true$ = /** @type {Sk.builtin.bool} */ (Object.create(Sk.builtin.bool.prototype, {v: {value: 1, enumerable: true}}));
-
-/**
- * Python bool False constant.
- * @type {Sk.builtin.bool}
- * @memberOf Sk.builtin.bool
- */
-Sk.builtin.bool.false$ = /** @type {Sk.builtin.bool} */ (Object.create(Sk.builtin.bool.prototype, {v: {value: 0, enumerable: true}}));
 
 /* Constants used for kwargs */
 
-// Sk.builtin.int_
```

Comment:
Yeah since doing int(“1”) is now handled completely differently. 

Before it would be dealt with by the applyOrSuspend architecture. Now it is handled by the cpython style architecture 
tp$call -> tp$new -> tp$init

at each stage the args, kwargs are passed and dealt with (mutable types typically handle args and kwargs in tp$init and immutable in tp$new). Every type object now gets type.prototype.tp$call method. 
It should make everything a lot cleaner.